### PR TITLE
add sound selection flow card ZW080

### PR DESCRIPTION
**Pull request by caseda**

### CODE REVIEW

File: drivers/ZW080/driver.js [hunk 1]
```diff
@@ -73,4 +73,34 @@ Homey.manager('flow').on('action.ZW080-turn_alarm_off', (callback, args) => {
...
\ No newline at end of file
+});
+
+Homey.manager('flow').on('action.ZW080-set_alarm', (callback, args) => {
+	const node = module.exports.nodes[args.device.token];
+	if (node &&
+		typeof node.instance.CommandClass.COMMAND_CLASS_CONFIGURATION !== 'undefined' &&
+		typeof args.volume !== 'undefined' &&
+		typeof args.sound !== 'undefined') {
+		const newValue = parseInt(args.volume) + parseInt(args.sound);
```
Comment (RobinBol) [inline drivers/ZW080/driver.js]: Maybe good practice to add a try/catch around the parseInt's and writeUIntBE(), just in case.

**Comment by jeroenvollenbrock** [comment]
Hi @caseda, Could you please have a look at the requested changes from @RobinBol so we can merge this PR as soon as possible?

**Comment by caseda** [comment]
Will do so this evening don't have time before that, also still waiting for a user that tested it, for why it wasn't working fully for him.

**Comment by caseda** [comment]
Ok, i changed it to the try->catch function, makes sense, saves a lot of checks you otherwiseneed to do manually.

sorry it took so long, was busy this weekend... and next time when i test sending parameters to a device... don't test it with a battery powered device 😅 been looking for why it never sends the configuration, though it's just because the device is just asleep...

**Pull request merged**